traits/src/lib.rs

```diff
@@ -16,13 +16,15 @@ pub use currency::{
 	LockIdentifier, MultiCurrency, MultiCurrencyExtended, MultiLockableCurrency, MultiReservableCurrency, OnReceived,
 };
 pub use data_provider::{DataFeeder, DataProvider, DataProviderExtended};
+pub use get_by_key::GetByKey;
```

Comment:
Do you need to expose `get_by_key_type` macro here?

Reply:
`#[macro_export]` exports the macro at crate root.